app/presenters/print/alert_presenter.rb

```diff
@@ -26,13 +31,13 @@ def to_s
 
     private
 
-    attr_reader :view_context
+    attr_reader :view_context, :toggle
     alias h view_context
 
     def build_status_content
       h.content_tag(:div, class: "image #{alert_class}") do
         alert_contents = [h.content_tag(:span, title, class: 'alert-title')]
-        alert_contents << h.wicked_pdf_image_tag('ic_red_tick.png') if status == :on
+        alert_contents << toggle_content if toggle_content
```

Comment:
Think we can just ignore the `if`. If that call returns nil it wont affect `safe_join`.